src/validate/validate_expression.ts

```diff
@@ -42,5 +43,11 @@ export function validateExpression(options: any): Array<ValidationError> {
         }
     }
 
-    return [];
+    const globalStateExpressionKeys = new Set<string>()
+
+    findGlobalStateExpressionKeys(expressionObj, globalStateExpressionKeys);
+
+    return Array.from(globalStateExpressionKeys).map((globalStateExpressionKey: string) => {
```

Comment:
Is there a better way to iterate over this set? This looks a bit cumbersome...

Reply:
Refactored, I think it's much better now